package.json

```diff
@@ -5,7 +5,9 @@
   "main": "lib/index.js",
   "scripts": {
     "prepublish": "npm run build",
-    "test": "npm run lint",
+    "jest": "[ `id -u` -ne 0 ] && NODE_ENV=test SILENCE_POLYGLOT_WARNINGS=1 jest --config test/jest-config.json --runInBand",
```

Comment:
I don't think we need to silence polyglot warnings and we also don't need [ `id -u` -ne 0 ] in this project